--- charts-original/templates/_node.tpl
+++ charts/templates/_node.tpl
@@ -8,10 +8,11 @@
   namespace: {{ .Values.node.namespaceOverride | default .Release.Namespace }}
   labels:
     {{- include "aws-ebs-csi-driver.labels" . | nindent 4 }}
-  {{- with .Values.node.daemonSetAnnotations }}
   annotations:
+    {{- include "aws-ebs-csi-driver.patchedAnnotations" . | nindent 4 }}
+    {{- with .Values.node.daemonSetAnnotations }}
     {{- toYaml . | nindent 4 }}
-  {{- end }}
+    {{- end }}
 spec:
   {{- if or (kindIs "float64" .Values.node.revisionHistoryLimit) (kindIs "int64" .Values.node.revisionHistoryLimit) }}
   revisionHistoryLimit: {{ .Values.node.revisionHistoryLimit }}
@@ -44,6 +45,7 @@
         {{- toYaml . | nindent 8 }}
         {{- end }}
       serviceAccountName: {{ .Values.node.serviceAccount.name }}
+      automountServiceAccountToken: true
       terminationGracePeriodSeconds: {{ .Values.node.terminationGracePeriodSeconds }}
       priorityClassName: {{ .Values.node.priorityClassName | default "system-node-critical" }}
       tolerations:
