--- charts-original/templates/_pod.tpl
+++ charts/templates/_pod.tpl
@@ -135,7 +135,8 @@
   {{- end }}
   volumes:
     - name: temp
-      emptyDir: {}
+      emptyDir:
+        sizeLimit: 256Mi
     {{- if ne $target "canary" }}
     - name: config
       {{- include "loki.configVolume" . | nindent 6 }}
@@ -197,7 +198,8 @@
       emptyDir: {}
       {{- end }}
     - name: sc-rules-temp
-      emptyDir: {}
+      emptyDir:
+        sizeLimit: 256Mi
     {{- end }}
     {{- if has $target (list "ruler" "backend" "single-binary") }}
     {{- range $dir, $_ := .Values.ruler.directories }}
