--- charts-original/templates/memcached/_memcached-statefulset.tpl
+++ charts/templates/memcached/_memcached-statefulset.tpl
@@ -53,6 +53,9 @@
         {{- end }}
     spec:
       serviceAccountName: {{ include "loki.serviceAccountName" (dict "ctx" $.ctx "component" $.ctx.Values.memcached "target" "memcached" ) }}
+      {{- if kindIs "bool" .automountServiceAccountToken }}
+      automountServiceAccountToken: {{ .automountServiceAccountToken }}
+      {{- end }}
       {{- if .priorityClassName }}
       priorityClassName: {{ .priorityClassName }}
       {{- end }}
